pcdsdevices/epics_motor.py

```diff
@@ -447,8 +447,6 @@ class PCDSMotorBase(EpicsMotorInterface):
     # paused and ready to resume on Go 'Paused', and to resume a move 'Go'.
     motor_spg = Cpt(EpicsSignal, '.SPG', kind='omitted')
 
-    tab_whitelist = ["spg_stop", "spg_pause", "spg_go"]
```

Comment:
I don't remember the origin of these- probably ok to remove without much more consideration

Reply:
I think few scientists would want to use these - agreed on the removal.